libavif.pc.cmake

```diff
@@ -0,0 +1,5 @@
+Name: ${PROJECT_NAME}
```

Comment:
It's customary to start with
```bash
prefix=@CMAKE_INSTALL_PREFIX@
exec_prefix=${prefix}/bin
libdir=${prefix}/lib
includedir=${prefix}/include
```
in case someone installs it to a place other than what's expected.

This is also where it would be helpful to use `@ONLY` so the end distributor or user can sed the `'s#prefix=.*#prefix=/MyPrefix#'` in case they want to install the end libraries in some place other than the one specified to cmake without needing to sed the rest of the lines